packages/tenant-discovery-nodejs/README.md

```diff
@@ -0,0 +1,236 @@
+# SuperTokens Plugin Tenant Discovery
+
+Automatically discover and route users to appropriate tenants based on their email domains.
+This plugin provides endpoints to infer tenant IDs from email domains and automatically assign users to the correct tenant during authentication.
+
+## Installation
+
+```bash
+npm install @supertokens-plugins/tenant-discovery-nodejs
+```
+
+## Quick Start
+
+### Backend Configuration
+
+Initialize the plugin in your SuperTokens backend configuration:
+
+```typescript
+import SuperTokens from "supertokens-node";
+import TenantDiscoveryPlugin from "@supertokens-plugins/tenant-discovery-nodejs";
+
+SuperTokens.init({
+  appInfo: {
+    // your app info
+  },
+  recipeList: [
+    // your other recipes
+  ],
+  plugins: [
+    TenantDiscoveryPlugin.init({
+      restrictedEmailDomains: ["tempmail.com", "10minutemail.com"], // Optional: additional domains to block
```

Comment:
I get what you meant, but the name/comment could be misleading upon first read: these aren't really restricted, and they are not blocked (e.g., they can sign in/up)